src/plugins/advanced_settings/public/management_app/components/form/form.test.tsx

```diff
@@ -19,7 +19,8 @@
 
```

Comment:
A general note to this file (not this PR): these test all seem to use `.toMatchSnapshot()` assertion. I'm not sure how useful it is, as those snapshots are large and only the original author really know what they *"should render"*. I would like to see specific assertions, like in the test case *"should render read-only when saving is disabled"* the assertion can specifically check for the read-only in specific element.